slack_sdk/http_retry/builtin_handlers.py

```diff
@@ -69,7 +73,7 @@ def prepare_for_next_attempt(
         if response is None:
             raise error
 
-        state.increment_current_attempt()
+        state.next_attempt_requested = True
```

Comment:
This should exist here but it was missing in this method (this bug affects only RateLimitedErrorRetryHandler)